README.md

```diff
@@ -135,18 +134,6 @@ changes. The glob matches files relative to the repository root.
       '**pyproject.toml'
 ```
 
-### API rate limit
```

Comment:
Please replace that section with explaining the input option (token used to get uv release artifacts, there is a default, why would I want to replace the default)